### ISSUE TITLE
problems everything for training and lora too

### ISSUE THREAD

**Issue by Mistermango23** [Linaqruf/kohya-trainer#325]
I wanted to prepare for the new Lora update and suddenly I got a huge bug picture. The result was balanced like this, I don't know if I did it wrong or the project was wrong.

Model: yiffymix v40

network_category: Lora

dim: 164
alpha: 164

optimizer_type: Dadaptation
unet_lr: 1
text_encoder_lr: 0.5
lr_scheduler: polynomial

![im_20231226102629_000_1897992832](https://github.com/Linaqruf/kohya-trainer/assets/111660652/d99f2738-b4d5-4570-a008-db54bca815a9)
![im_20231226102617_000_1603694585](https://github.com/Linaqruf/kohya-trainer/assets/111660652/0ee793c0-9492-411c-bcde-9a8f60d1705a)

![err2](https://github.com/Linaqruf/kohya-trainer/assets/111660652/683e825d-3747-4689-b513-fc6650ba3541)
![err1](https://github.com/Linaqruf/kohya-trainer/assets/111660652/5efa4335-5a07-4cbd-b4d0-7f30661a13b1)

**Comment by Mistermango23** [comment]
Sorry! my mistake
